msbot/mslib.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import json
+from spoiler import Spoiler
+
+
+
+def getCardsBySet(setname):
+       	"""
+       	:type setname: String
+       	:rtype: List[Spoiler]
+       	"""
+	url = 'http://mythicspoilerapi.dungeonmastering.net/APIv2/cards/by/set'
+	payload = {'key': 'ZWdneml0QGdtYWlsLmNvbQ==', 'param': setname}
```

Comment:
Both the url and api key should be declared as constants at the beginning of this file.

e.g.
```
 BASE_API_URL = 'http://mythicspoilerapi.dungeonmastering.net/APIv2'
 MS_API_KEY = 'ZWdneml0QGdtYWlsLmNvbQ=='
```

although ideally these api keys should be environment varaibles